src/Confluent.Kafka/Consumer.cs

```diff
@@ -620,6 +620,27 @@ public Metadata GetMetadata(bool allTopics, TimeSpan timeout)
         /// </summary>
         public Metadata GetMetadata(bool allTopics)
             => consumer.GetMetadata(allTopics);
+
+        /// <summary>
+        ///     Adds one or more brokers to the Consumer's list of initial
+        ///     bootstrap brokers. 
+        ///
+        ///     Note: Additional brokers are discovered automatically as 
+        ///     soon as the Consumer connects to a broker by querying the 
+        ///     broker metadata. Calling this method is only required in 
+        ///     some scenarios where the address of all brokers in the 
+        ///     cluster changes.
+        /// </summary>
+        /// <param name="brokers">
+        ///     Coma-separated list of brokers in the same format as 
+        ///     the bootstrap.server configuration parameter.
+        /// </param>
+        /// <example>
+        ///     AddBrokers("broker1:9092,broker2")
+        ///     AddBrokers("SSL://broker3:9000,ssl://broker2")
```

Comment:
the protocol notation is unnecessary since the protocol to use is set by `security.protocol`.